--- dovecot-2.2.16/src/plugins/quota/Makefile.am.0001~	2014-09-10 12:38:58.000000000 +0200
+++ dovecot-2.2.16/src/plugins/quota/Makefile.am	2015-03-13 16:11:42.295338661 +0100
@@ -47,7 +47,7 @@ quota_common_objects = \
 
 lib10_quota_plugin_la_SOURCES = $(quota_dist_sources)
 nodist_lib10_quota_plugin_la_SOURCES = $(RQUOTA_XDR)
-lib10_quota_plugin_la_LIBADD = $(QUOTA_LIBS)
+lib10_quota_plugin_la_LIBADD = $(QUOTA_LIBS) -ltirpc
 
 doveadm_module_LTLIBRARIES = \
 	lib10_doveadm_quota_plugin.la
--- dovecot-2.2.16/src/plugins/quota/Makefile.in.0001~	2015-03-13 16:11:42.295338661 +0100
+++ dovecot-2.2.16/src/plugins/quota/Makefile.in	2015-03-13 16:12:13.346675404 +0100
@@ -275,7 +275,7 @@ LD = @LD@
 LDAP_LIBS = @LDAP_LIBS@
 LDFLAGS = @LDFLAGS@
 LIBCAP = @LIBCAP@
-LIBDOVECOT = @LIBDOVECOT@
+LIBDOVECOT = @LIBDOVECOT@ -ltirpc
 LIBDOVECOT_COMPRESS = @LIBDOVECOT_COMPRESS@
 LIBDOVECOT_DEPS = @LIBDOVECOT_DEPS@
 LIBDOVECOT_DSYNC = @LIBDOVECOT_DSYNC@
