--- dovecot-ce-2.3.0/src/plugins/quota/Makefile.am.0001~	2017-12-22 14:53:41.000000000 +0100
+++ dovecot-ce-2.3.0/src/plugins/quota/Makefile.am	2017-12-26 01:17:53.988281433 +0100
@@ -55,7 +55,7 @@ quota_common_objects = \
 
 lib10_quota_plugin_la_SOURCES = $(quota_dist_sources)
 nodist_lib10_quota_plugin_la_SOURCES = $(RQUOTA_XDR)
-lib10_quota_plugin_la_LIBADD = $(QUOTA_LIBS)
+lib10_quota_plugin_la_LIBADD = $(QUOTA_LIBS) -ltirpc
 
 doveadm_module_LTLIBRARIES = \
 	lib10_doveadm_quota_plugin.la
--- dovecot-ce-2.3.0/src/plugins/quota/Makefile.in.0001~	2017-12-26 01:17:53.988281433 +0100
+++ dovecot-ce-2.3.0/src/plugins/quota/Makefile.in	2017-12-26 01:20:45.727762351 +0100
@@ -335,7 +335,7 @@ LDFLAGS = @LDFLAGS@
 LD_NO_WHOLE_ARCHIVE = @LD_NO_WHOLE_ARCHIVE@
 LD_WHOLE_ARCHIVE = @LD_WHOLE_ARCHIVE@
 LIBCAP = @LIBCAP@
-LIBDOVECOT = @LIBDOVECOT@
+LIBDOVECOT = @LIBDOVECOT@ -ltirpc
 LIBDOVECOT_COMPRESS = @LIBDOVECOT_COMPRESS@
 LIBDOVECOT_DEPS = @LIBDOVECOT_DEPS@
 LIBDOVECOT_DSYNC = @LIBDOVECOT_DSYNC@
