--- dovecot-2.2.26.0/src/lib-fts/Makefile.am.omv~	2016-10-31 17:36:08.324709127 +0100
+++ dovecot-2.2.26.0/src/lib-fts/Makefile.am	2016-10-31 17:36:25.734708164 +0100
@@ -9,6 +9,7 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/src/lib-mail \
 	-I$(top_srcdir)/src/lib-fts \
 	$(LIBEXTTEXTCAT_CFLAGS) \
+	-I/usr/include/libstemmer \
 	$(LIBICU_CFLAGS) \
 	-DUDHRDIR=\""$(top_srcdir)/src/lib-fts"\" \
 	-DDATADIR=\"$(pkgdatadir)\" \
--- dovecot-2.2.26.0/src/plugins/fts-lucene/Makefile.am.omv~	2016-10-31 17:47:01.398673003 +0100
+++ dovecot-2.2.26.0/src/plugins/fts-lucene/Makefile.am	2016-10-31 17:47:11.605672439 +0100
@@ -6,7 +6,8 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/src/lib-index \
 	-I$(top_srcdir)/src/lib-storage \
 	-I$(top_srcdir)/src/plugins/fts \
-	-I$(top_srcdir)/src/doveadm
+	-I$(top_srcdir)/src/doveadm \
+	-I/usr/include/libstemmer
 
 AM_CXXFLAGS = \
 	$(CLUCENE_CFLAGS) \
